main.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
```

Comment:
Awesome job parsing out your main logic into its own file that each page included! This was a great Idea!